### PR TITLE
assorted: optionally add russian tag to titles. resolves #12710

### LINKED ISSUE #12710: [req]: Aviability to add a text to entry's name
### Is there already a request for your feature?

- [X] I have checked older issues, open and closed

### Is your feature request related to a problem? Please describe.

The problem is russian torrent trackers do not follow torrent standards. It can be understood - there's no need to write "RUS" or "Russian" sound to name if 99% names are in Russian. But when Radarr tries to parse smth like `Южный Парк: После COVID’а / South Park: Post COVID (2021)` it has no language mentioned and parsed as original-language by Radarr and not grabbed cause i ask Russian but get English. I guess the same problem can be found in any non-English tracker which is oriented to national audio.

Firstly i've asked Radarr for help but they've answered that is not a Radarr issue: Torrent was created with no sound info -> Tracker does not add it in RSS -> Jacket doesn't add any info -> there's no info. For now the only way to fix the problem in such Russian trackers is to choose "Any" language in Radarr so English will be grabbed too. 

The exact quote of solution they offer:
> Just tell them they need to add RUS or RUSSIAN to the release names for that indexer

### Describe the solution you'd like

The feature request is to add the text field for indexer and glue a text from there to the torrent name everywhere / in the feed so that Radarr could get search results like this:
`RUS Южный Парк: После COVID’а / South Park: Post COVID (2021) BDrip`
`RUS Южный Парк: После COVID’а / South Park: Post COVID (2021) 720p`
`RUS Южный Парк: После COVID’а / South Park: Post COVID (2021) CamRip медвед`
if the field has a "RUS " input.

### Describe alternatives you've considered

As an alternative i can see that Jackett does already have a tags system. Optional glueing the tags to torrent name colud be an alternative sollution.

Another way is to tune the exact russian indexers to hardcode adding "RUS" to their entry names.

### PR DESCRIPTION
**Pull request by ilike2burnthing**
resolves #12710

@garfield69 is there a more elegant or targeted way of doing this?

### CODE REVIEW

**Comment by garfield69** [comment]
>is there a more elegant or targeted way of doing this?

you mean a method that does not rely on a config but that auto-detect if the title has Russian lettering and adds the tag ?
sure.
how about this example, using nonameclub as source:
- in the field section, before the title block:
```yaml
    title_is_rus:
      selector: a[href^="viewtopic.php?t="] > b
      filters:
        - name: re_replace
          args: ["[^А-Яа-яЁё]+", ""]
```
- and then in the title block, after all the other filters, the following:
``` yaml
        - name: append
          args: "{{ if .Result.title_is_rus }} - RUSSIAN{{ else }}{{ end }}"
```

File: src/Jackett.Common/Definitions/rutor.yml [hunk 1]
```diff
@@ -70,6 +74,11 @@ search:
   fields:
     category:
       text: 3
+    title_is_rus:
+      selector: a[href^="viewtopic.php?t="] > b
+      filters:
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rutor.yml:79]: you need to use the selector for the title as below, this is not nonameclub ;-)

**Review by garfield69 — CHANGES_REQUESTED** [review]

File: src/Jackett.Common/Definitions/rutor.yml (hunk 1 above)
Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/rutor.yml:79]: Well that would just be sensible... 🤦 brb

**Review by garfield69 — APPROVED** [review]
tested rutor LGTM
unable to test NNC its timing out for me  ATM

**Pull request merged**